design-system/Button.svelte

```diff
@@ -245,7 +264,7 @@
   class="button typo-text-bold {buttonClass}"
   {title}
   {disabled}
-  on:click
+  on:click|stopPropagation
```

Comment:
I wonder if we shouldn't let the callsite decide on what modifiers to use. 🤔 
This should work `<Button on:click|stopPropagation={} />`, no?

Reply:
Nope, you can only do modifiers on dom elements.

Reply:
Right. I guess this should be fine then. We probably don't want button presses to propagate anyway.